perfectscale/README.md

```diff
@@ -0,0 +1,43 @@
+# Agent Check: Perfectscale
+
+## Overview
+
+
+PerfectScale is a comprehensive cloud-agnostic solution that enables teams with continuous and autonomous Kubernetes optimization capabilities. Our main goal is to empower teams to effortlessly manage their infrastructure, make quick, data-driven decisions, and reach peak performance at the lowest possible cost.
+
+PerfectScale monitors resiliency metrics throughout your Kubernetes environment. When PerfectScale identifies under-provisioned or misconfigured workloads, it promptly pushes an alert to the Datadog dashboard detailing the issue type (such as OOM, CPU-Throttling, Eviction, and more) along with its precise location (cluster, namespace, workload, and container).
```

Comment:
Can you add what specific data the integration monitors? For example, if you are only sending in metrics please add that this integration sends in <type of metric> metrics to Datadog.

Reply:
Done